sites/public/pages/applications/review/summary.tsx

```diff
@@ -40,7 +40,12 @@ const ApplicationSummary = () => {
           </h2>
         </div>
 
-        <FormSummaryDetails application={application} listing={listing} editMode />
+        <FormSummaryDetails
+          application={application}
+          listing={listing}
+          hidePreferences={listing.listingPreferences.length === 0}
```

Comment:
Seems like we could do this inside FormSummaryDetails rather than add a prop

Reply:
There's at least one other case where FormSummaryDetails is called with `hidePreferences` passed in, and in that case, it just passes true to always hide them.